### ISSUE TITLE
Make modified frcmod to SMIRFF XML converter

### ISSUE THREAD

**Issue by davidlmobley** [open-forcefield-group/smarty#118]
@cbayly13 is attempting to put parm@frosst into SMIRFF format during his final time here. To avoid him having to mess with hand-typing the XML, the initial plan is for him to just go through the relevant AMBER parameter files and put the relevant SMIRKS pattern on each parameter line instead of the atom types.

I will implement a converter tool which converts these to SMIRFF XML files. 

The format of the files I'll be looking at are (artificial example)
- atom types:

```
[#1:1]           0.6000  0.0157            !Ferguson base pair geom.; H
[#1:1]~[#8]          0.0000  0.0000             OPLS Jorgensen, JACS,110,(1988),1657
...
```
- bonds:

```
[#6:1]~[#6:2]    310.0    1.525       Junmei et al, 1999; C-C
[#6:1]~[#6a:2]  469.0    1.409       JCC,7,(1986),230; (not used any more in TYR); C-CA
...
```
- angles:

```
[#6:1]~[#6:2]~[#8:3]     80.0      120.00    Junmei et al, 1999 acrolein; C -C -O
[#6:1]~[#6:2]~[#8H1:3]    80.0      120.00    Junmei et al, 1999
...
```
- torsions (note multiple lines which need merging):

```
[*:1]~[#6:2]~[#6:3]~[*:4]    4   14.50        180.0             2.         Junmei et al, 1999; X -C -C -X 
[*:1]~[#6:2]~[#6a:3]~[*:4]    4   14.50        180.0             2.         intrpol.bsd.on C6H6; X -C -CA-X
...
[#6X4:1]-[#6X4:2]-[#8X2:3]-[#1:4] 1    0.16          0.0            -3.         Junmei et al, 1999 HO-OH-CT-CT
[#6X4:1]-[#6X4:2]-[#8X2:3]-[#1:4] 1    0.25          0.0             1.         Junmei et al, 1999 HO-OH-CT-CT
...
```
- impropers:

```
[*:1]~[*:2]~[#6:3]~[#8:4]          10.5         180.          2.           JCC,7,(1986),230; improper X -X -C -O
...
```

Parm@frosst is available here (http://www.ccl.net/cca/data/parm_at_Frosst/) and is a supplement to parm99 which is available in standard AMBER distributions. 

I'll probably put this in a `utilities` directory called `convert_frosst` or something along those lines, and will populate also with Bayly's source files when he has them.

**Comment by jchodera** [comment]
Sounds good!

**Comment by davidlmobley** [comment]
Closed by #120
